### PR TITLE
Add IBJ and Ice Beam Strats in Norfair East

### PR DESCRIPTION
**Pull request by kjbranch**
Single Chamber and Upper Norfair Farming Room

### CODE REVIEW

File: region/norfair/east.json [hunk 1]
```diff
@@ -3585,6 +3643,26 @@
...
+                },
+                {
+                  "name": "Frozen Enemies",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canUseFrozenEnemies",
+                    {"heatFrames": 1200}
+                  ],
+                  "note": "Wait for the Alcoon to walk off the edge. Use it and the Multiviola as platforms."
```
Comment (osse101) [inline region/norfair/east.json:3665]: For this dboost either have the alcoon walkinto you at the height of your jump, or get iframes first and jump through it.
Harder to do than the others.

File: region/norfair/east.json [hunk 2]
```diff
@@ -3647,6 +3725,25 @@
...
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canIBJ",
+                    {"heatFrames": 1500}
+                  ],
+                  "note": "Kill the enemies, then IBJ."
+                },
+                {
+                  "name": "Frozen Enemies",
```
Comment (osse101) [inline region/norfair/east.json:3740]: Alcoon as it jumps out of the ground dboost, or multiviola dboost.

**Review by osse101 — COMMENTED** [review]
Going to trust the heatframe numbers.

**Comment by kjbranch** [comment]
Some of this feedback can be used in #514

**Pull request merged**